src/tls/Makefile

```diff
@@ -1,32 +1,25 @@
 EXTRACT='Nothing'
-FLAVOR=generic
+FLAVOR?=Model
 
-all:
-	rm -f .depend.OCaml .depend.Kremlin
-	$(MAKE) ocaml-all kremlin-all
-
-clean-depend:
-	rm -rf .depend.OCaml .depend.Kremlin
-
-ocaml-all kremlin-all: clean-depend
-
-include Makefile.common
+all: ocaml-all kremlin-all
 
 ocaml-%:
 	$(MAKE) -f Makefile.OCaml $*
 
 kremlin-%:
 	$(MAKE) -f Makefile.Kremlin $*
 
+model-% verify-%:
+	$(MAKE) -f Makefile.Model $*
+
 quic-%:
 	$(MAKE) -C ../../apps/quicMinusNet $* -k
 
 test: kremlin-test ocaml-test quic-test
 
-clean: kremlin-clean ocaml-clean quic-clean
+clean: ocaml-clean kremlin-clean quic-clean model-clean
 	rm -rf $(EXTRACT_DIR)/Kremlin $(EXTRACT_DIR)/OCaml $(EXTRACT_DIR)/copied \
-	$(EXTRACT_DIR)/Kremlin-{Internal-Test,Library}/{,*/}*.{c,h,o} \
-	.depend.$(FLAVOR) *.checked.lax 
+	*.checked *.checked.lax
```

Comment:
I suspect the removal of `*.checked` is not very effective since by default they go in .cache.$(FLAVOR), right?

Reply:
Yes, and those are cleaned up in the respective Makefile.*'s.

Reply:
shall we remove `*.checked *.checked.lax` from the invocation above then? launching `rm -rf *.checked` when there is no match seems confusing to me